From 76946a8c03488d74e21f1ca39b640bc8c07ba7c3 Mon Sep 17 00:00:00 2001
From: Cyril Bur <cyrilbur@gmail.com>
Date: Thu, 5 May 2016 11:27:15 +1000
Subject: [PATCH 37/56] arm/dts: Remove mac hardware checksum from Firestone
 BMC Board

Hardware checksumming doesn't work.

Without this patch pings work but more complex network interactions like SSH
fail.

Signed-off-by: Cyril Bur <cyrilbur@gmail.com>
Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 arch/arm/boot/dts/aspeed-bmc-opp-firestone.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-bmc-opp-firestone.dts b/arch/arm/boot/dts/aspeed-bmc-opp-firestone.dts
index 379b561..51b03b7 100644
--- a/arch/arm/boot/dts/aspeed-bmc-opp-firestone.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-opp-firestone.dts
@@ -41,6 +41,9 @@
 #include "aspeed-bmc-opp-flash-layout.dtsi"
 			};
 		};
+		mac0: ethernet@1e660000 {
+			no-hw-checksum;
+		};
 		spi@1e630000 {
 			reg = < 0x1e630000 0x18
 				0x30000000 0x02000000 >;
-- 
1.9.1

